src/main/java/net/runelite/data/dump/wiki/NpcStatsDumper.java

```diff
@@ -0,0 +1,259 @@
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package net.runelite.data.dump.wiki;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Sets;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.cache.NpcManager;
+import net.runelite.cache.definitions.NpcDefinition;
+import net.runelite.cache.fs.Store;
+import net.runelite.cache.util.Namer;
+import net.runelite.data.App;
+import net.runelite.data.dump.MediaWiki;
+import net.runelite.data.dump.MediaWikiTemplate;
+
+@Slf4j
+public class NpcStatsDumper
+{
+	@Value
+	@Builder
+	private static final class NpcStats
+	{
+		private final Integer hitpoints;
+		private final Integer combatLevel;
+		private final Integer slayerLevel;
+
+		private final Integer attackLevel;
+		private final Integer strengthLevel;
+		private final Integer defenceLevel;
+		private final Integer rangeLevel;
+		private final Integer magicLevel;
+
+		private final Integer stab;
+		private final Integer slash;
+		private final Integer crush;
+		private final Integer range;
+		private final Integer magic;
+
+		private final Integer stabDef;
+		private final Integer slashDef;
+		private final Integer crushDef;
+		private final Integer rangeDef;
+		private final Integer magicDef;
+
+		private final Integer bonusAttack;
+		private final Integer bonusStrength;
+		private final Integer bonusRangeStrength;
+		private final Integer bonusMagicDamage;
+
+		private final Boolean poisonImmune;
+		private final Boolean venomImmune;
+	}
+
+	private static final NpcStats DEFAULT = NpcStats.builder().build();
+
+	public static void dump(final Store store, final MediaWiki wiki) throws IOException
+	{
+		final File out = new File("runelite/runelite-client/src/main/resources/");
+		out.mkdirs();
+
+		log.info("Dumping npc stats to {}", out);
+
+		final NpcManager npcManager = new NpcManager(store);
+		npcManager.load();
+
+		final Set<Integer> npcIds = getAttackableNpcIds(npcManager);
+		final Set<Integer> foundIds = new HashSet<>();
+
+		log.info("Dumping {} npc stats...this may take a while", npcIds.size());
```

Comment:
this may take a while... is not necessary, just use exactly same format as other dumpers do